core/rend/TexCache.cpp

```diff
@@ -212,9 +212,11 @@ void BaseTextureCacheData::PrintTextureName()
 	if (tsp.FilterMode != 0)
 		strcat(str, " Bilinear");
 
-	sprintf(str + strlen(str), " %dx%d @ 0x%X", 8 << tsp.TexU, 8 << tsp.TexV, tcw.TexAddr << 3);
+	size_t len = strlen(str);
+
+	snprintf(str + len, sizeof(str) - len, " %dx%d @ 0x%X", 8 << tsp.TexU, 8 << tsp.TexV, tcw.TexAddr << 3);
 	std::string id = GetId();
-	sprintf(str + strlen(str), " id=%s", id.c_str());
+	snprintf(str + len, sizeof(str) - len, " id=%s", id.c_str());
```

Comment:
`len` must first be updated to the new string length or this call will overwrite the previous snprintf.